tests/nest_desktop_tests/nest_desktop_code_generator_test.py

```diff
@@ -0,0 +1,64 @@
+# -*- coding: utf-8 -*-
```

Comment:
The test is currently not being run in CI. Could you add a separate stage for it, just like we have for the Python code generator and the compartmental code generator?

@babsey: is there any integration test we could do with NEST Desktop in CI?

Reply:
Unfortunately, I do not have any kind of testing suites of NEST Desktop in CI.